.github/workflows/build.yml

```diff
@@ -155,6 +155,8 @@ jobs:
 
     - name: Test MacOS
       if: matrix.os == 'macos-latest'
+      env:
+        OBJC_DISABLE_INITIALIZE_FORK_SAFETY: 'YES'
```

Comment:
Could you comment in-code on why this was added?

Reply:
I will add a comment but I am sure it is not the right fix for the issue that required adding this line. 

If the line is removed. On macos the following error appears:
```
objc[67570]: +[__NSPlaceholderDate initialize] may have been in progress in another thread when fork() was called.
objc[67570]: +[__NSPlaceholderDate initialize] may have been in progress in another thread when fork() was called. We cannot safely call it or ignore it in the fork() child process. Crashing instead. Set a breakpoint on objc_initializeAfterForkError to debug.
```

For reference, this is where I found the fix for the error above: https://stackoverflow.com/questions/50168647/multiprocessing-causes-python-to-crash-and-gives-an-error-may-have-been-in-progr

I could not find the underlying issue that was causing the error and I found it a bit strange because we are using `frokserver` when starting the process that spins up perfect.  

Also, the fun part is that the error appears intermittently on macOS :)

Reply:
Seems clear from the SO answer that you're laxing security introduced in High Sierra?

So maybe the comment could be:

`# Relax multithreading security measures taken by macOS High Sierra `

Or something

Reply:
Perfect!